src/main/java/com/redhat/exhort/integration/providers/osvnvd/OsvNvdIntegration.java

```diff
@@ -60,6 +60,17 @@ public void configure() throws Exception {
       .to(vertxHttp("{{api.osvnvd.host}}"))
       .transform().method(responseHandler, "responseToIssues");
 
+    from(direct("osvNvdHealthCheck"))
+        .routeId("osvNvdHealthCheck")
```

Comment:
Please, try to follow the same format as in the other routes.

Reply:
@ruromero But i don't have a token in this osvNvd Provider, so it's impossible to use the same format, so which format are you expecting?
or you meant something else?
Please give example what is expected for clarification.

Reply:
I mean indenting. See that the previous route uses 2 spaces and children nodes are indented.
```
from(..)
  .routeId(..)
  .circuitBreaker()
    .faultToleranceConfiguration()
...
```

Reply:
clear, Fixed.